src/main/java/com/futurcollars/lesson8/zad7/DayOfWeek.java

```diff
@@ -0,0 +1,5 @@
+package com.futurcollars.lesson8.zad7;
+
+public enum DayOfWeek {
+    PONIEDZIAŁEK, WTOREK, ŚRODA, CZWARTEK, PIĄTEK, SOBOTA, NIEDZIELA
```

Comment:
używanie polskich znaków poza wartością stringa jest bardzo problematyczne, dużo lepiej byłoby dać enuma po angielsku a zwracać polskie nazwy w metodzie, ale jak działa to działą :p